pkg/operatorclient/operatorclient.go

```diff
@@ -37,6 +38,19 @@ func (c OperatorClient) GetOperatorState() (*operatorv1.OperatorSpec, *operatorv
 	return &instance.Spec.OperatorSpec, &instance.Status.OperatorStatus, instance.ResourceVersion, nil
 }
 
+func (c OperatorClient) IsOperatorInstalled(name string) (bool, error) {
```

Comment:
`Operator` in `OperatorClient` refers to CSO, not CSI driver operator. `IsOperatorInstalled` does not really belong here. Use `driver_starter.go` or util.go in the same dir.

Reply:
Done.